srml/support/procedural/src/lib.rs

```diff
@@ -79,6 +79,17 @@ use proc_macro::TokenStream;
 ///   If the keys are not trusted (e.g. can be set by a user), a cryptographic `hasher` such as
 ///   `blake2_256` must be used. Otherwise, other values in storage can be compromised.
 ///
+///    Once read from the trie (e.g. through an RPC call), a tuple of `(T, Linkage<K>)` is returned,
```

Comment:
First, I would not mention the trie at all, we are just talking about storage here.
Second, I'm not sure this is the right place. Here it should be documented what kind of syntax the macro supports and not how the data is encoded in the storage. This should probably be moved above the trait.

Reply:
The primary reason that I placed it here is that this page is --in some way-- the only aggregated, well document crate that we have about storage (you also sent it to me when I had some questions). I assumed anyone wanting to know this would have a high chance of finding it here. 

Otherwise, I agree that the context of the new paragraph is different.